README.md

```diff
@@ -1 +1,18 @@
-# Test Automation Samples
\ No newline at end of file
+# Test Automation Samples
+
+Project build with gradle.
+Simple console app that give you possibility to simulate 
+behaviour of real animal shelter. 
+
+I've used facade pattern for that project 
```

Comment:
Avoid using "I" in **README.md**. People usually know who has created this project.